src/test/java/com/puppycrawl/tools/checkstyle/checks/indentation/IndentationCheckTest.java

```diff
@@ -2929,6 +2929,20 @@ public void testIndentationLineBreakVariableDeclaration()
         verifyWarns(checkConfig, fileName, expected);
     }
 
+    @Test
+    public void testMethodCallLineWarpWithArrayComponents()
```

Comment:
Issue was reported with `google_checks.xml`, please make a test case with this exact configuration:
https://github.com/checkstyle/checkstyle/blob/63d2e5cfea074bf200353b496cd0bfebe9b4ca97/src/main/resources/google_checks.xml#L249-L256

Also, it is good to keep properties in input file and properties in UT in the same order :)